docs/overview.asciidoc

```diff
@@ -24,11 +24,9 @@ More documentation is hosted in https://github.com/elastic/elasticsearch-ruby[Gi
 
 The `elasticsearch` gem combines two separate Rubygems:
 
-* https://github.com/elastic/elastic-transport-ruby/[`elastic-transport`]
-provides an HTTP Ruby client for connecting to the {es} cluster.
+* https://github.com/elastic/elastic-transport-ruby/[`elastic-transport`] - provides an HTTP Ruby client for connecting to the {es} cluster. See the documentation: <<transport>>
```

Comment:
Nit. It is recommended to use `refer to` over `see` for availability purposes (for example, if somebody uses a screen reader to access the content on the page).

```suggestion
* https://github.com/elastic/elastic-transport-ruby/[`elastic-transport`] - provides an HTTP Ruby client for connecting to the {es} cluster. Refer to the documentation: <<transport>>
```